torch/autograd/forward_ad.py

```diff
@@ -125,13 +127,18 @@ def make_dual(tensor, tangent, *, level=None):
 
 
 class UnpackedDualTensor(_UnpackedDualTensor):
-    r"""Namedtuple returned by :func:`unpack_dual` containing the primal and tangent components of the dual tensor.
-    See :func:`unpack_dual` for more details."""
+    r"""Named tuple returned by :func:`unpack_dual` containing the primal and tangent components of the dual tensor.
```

Comment:
NameTuple should probably stay as one word, as it's a Python class.

Reply:
Alright.